tests/unit/test_echostream.py

```diff
@@ -20,19 +20,8 @@ def test_simple_zmqlet():
         '--socket-out', 'PUSH_CONNECT',
         '--timeout-ctrl', '-1'])
 
-    args2 = set_pea_parser().parse_args([
-        '--host-in', '0.0.0.0',
-        '--host-out', '0.0.0.0',
-        '--port-in', '12347',
-        '--port-out', '12346',
-        '--socket-in', 'PULL_BIND',
-        '--socket-out', 'PUSH_BIND',
-        '--uses', '_logforward',
-        '--timeout-ctrl', '-1'
-    ])
-
     logger = logging.getLogger('zmq-test')
-    with BasePea(args2) as z1, Zmqlet(args, logger) as z:
+    with Zmqlet(args, logger) as z:
```

Comment:
I think this may be on putpose to test ifnthey can work in parallel

Reply:
okay, makes sense. I put it back and removed just the z1 parameter since it is not used any further